tests/by-util/test_base64.rs

```diff
@@ -40,6 +40,49 @@ fn test_encode_repeat_flags_later_wrap_15() {
         .stdout_only("aGVsbG8sIHdvcmx\nkIQ==\n"); // spell-checker:disable-line
 }
 
+// Different behavior between GNU 9.4 & 9.5
+// See GNU 8c735f65857ea17cdc0218b3fa3223769c3d8b9c
```

Comment:
To me this looks like an irrelevant comment because we always try to match the behavior of the latest GNU version.

Reply:
good point, i will remove it